NEWS

```diff
@@ -55,6 +55,43 @@ included in the vX.Y.Z section and be denoted as:
    (** also appeared: A.B.C)  -- indicating that this item was previously
                                  included in release version vA.B.C.
 
+3.1.4 -- February, 2019
+-----------------------
+
+- Fix compile error when configured with --enable-mpi-java and
+  --with-devel-headers.  Thanks to @g-raffy for reporting the issue
+  (** also appeared: v3.0.4).
+- Only use hugepages with appropriate permissions.  Thanks to Hunter
+  Easterday for the fix.
+- Fix possible floating point rounding and division issues in OMPIO.
```

Comment:
Those are not floating point rounding issues (this is just the implementation detail causing them), but they are
- crashes and
- data corruption

described in #6285 with currently four PRs for fixes (see PR description for full list).

Reply:
Done.